core/cont/src/TClassTable.cxx

```diff
@@ -255,6 +256,20 @@ TClassTable::~TClassTable()
    delete fgIdMap; fgIdMap = 0;
 }
 
+/// Return true is the table exist.
+/// If the table does not exist but the delayed list does, then
+/// create the table and return true.
+inline Bool_t TClassTable::CheckClassTableInit() {
+   if (!gClassTable || !fgTable) {
+      if (GetDelayedAddClass().size()) {
+         new TClassTable;
```

Comment:
Shouldn't you copy over the classes from the temporary table here?

Reply:
Yes.  This is done in the TClassTable constructor.